src/mono/mono/mini/interp/mintops.def

```diff
@@ -85,6 +85,7 @@ OPDEF(MINT_LDSFLD_R4, "ldsfld.r4", 4, 1, 0, MintOpUShortInt)
 OPDEF(MINT_LDSFLD_R8, "ldsfld.r8", 4, 1, 0, MintOpUShortInt)
 OPDEF(MINT_LDSFLD_O, "ldsfld.o", 4, 1, 0, MintOpUShortInt)
 OPDEF(MINT_LDSFLD_VT, "ldsfld.vt", 5, 1, 0, MintOpTwoShorts)
+OPDEF(MINT_LDSFLD_VT_W, "ldsfld.vt.w", 8, 1, 0, MintOpTwoInts)
```

Comment:
We use the VT suffix for opcodes operating on MINT_TYPE_VT. We should drop it for these opcodes.